File: Antibody.CareToKnowPro.CRM/ClientApp/src/app/model/loginProfileRole.ts
import { ILoginProfile as LoginProfile } from './loginProfile';
import { IRole as Role } from './role';

export interface ILoginProfileRole { 
    loginProfileRoleId?: number;
    loginProfileId?: number;
    roleId?: number;
    status?: string;
    loginProfile?: LoginProfile;
    role?: Role;
}

File: Antibody.CareToKnowPro.CRM/Models/LoginProfileRole.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace Antibody.CareToKnowPro.CRM.Models
{
    public partial class LoginProfileRole
    {
        public int LoginProfileRoleId { get; set; }
        public int? LoginProfileId { get; set; }
        public int? RoleId { get; set; }
        [StringLength(10)]
        public string Status { get; set; }

        [ForeignKey("LoginProfileId")]
        [InverseProperty("LoginProfileRole")]
        public virtual LoginProfile LoginProfile { get; set; }
        [ForeignKey("RoleId")]
        [InverseProperty("LoginProfileRole")]
        public virtual Role Role { get; set; }
    }
}
